src/main/resources/data/bloodmagic/patchouli_books/guide/ru_ru/templates/crafting_arc.json

```diff
@@ -0,0 +1,120 @@
+{
+  "processor": "wayoftime.bloodmagic.compat.patchouli.processors.ARCProcessor",
+  "components": [
+    {
+      "type": "header",
+      "text": "#heading",
+      "x": -1,
+      "y": -6
+    },
+    {
+      "type": "text",
+      "text": "#text",
+      "x": 0,
+      "y": 51
+    },
+    {
+      "type": "item",
+      "item": "#fluid_input",
+      "x": 6,
+      "y": 26
+    },
+    {
+      "type": "item",
+      "item": "#fluid_output",
+      "x": 94,
+      "y": 7
+    },
+    {
+      "type": "image",
+      "image": "bloodmagic:textures/gui/patchouli_book/crafting.png",
+      "x": 2,
+      "y": 4,
+      "u": 0,
+      "v": 198,
+      "texture_width": 128,
+      "texture_height": 256,
+      "width": 112,
+      "height": 43
+    },
+    {
+      "type": "item",
+      "item": "bloodmagic:alchemicalreactionchamber",
+      "x": 94,
+      "y": 26
+    },
+    {
+      "type": "item",
+      "item": "#input",
+      "x": 6,
+      "y": 7
+    },
+    {
+      "type": "item",
+      "item": "#tool",
+      "x": 31,
+      "y": 26
+    },
+    {
+      "type": "item",
+      "item": "#output1",
+      "x": 56,
+      "y": 7
+    },
+    {
+      "type": "item",
+      "item": "#output2",
+      "x": 75,
+      "y": 7
+    },
+    {
+      "type": "text",
+      "text": "#chance2#%",
+      "guard": "#show_chance2",
+      "x": 75,
+      "y": 6
+    },
+    {
+      "type": "item",
+      "item": "#output3",
+      "x": 56,
+      "y": 26
+    },
+    {
+      "type": "text",
+      "text": "#chance3#%",
+      "guard": "#show_chance3",
+      "x": 56,
+      "y": 25
+    },
+    {
+      "type": "item",
+      "item": "#output4",
+      "x": 75,
+      "y": 26
+    },
+    {
+      "type": "text",
+      "text": "#chance4#%",
+      "guard": "#show_chance4",
+      "x": 75,
+      "y": 25
+    },
+    {
+      "type": "tooltip",
+      "guard": "#show_fluid_tooltip",
+      "tooltip": ["Input: #tooltip_fluid_input#", "Output: #tooltip_fluid_output#"],

```

Comment:
This appears in a tooltip whenever an ARC recipes include a fluid component.  Need to translate "Input:" and "Output".  

The fluid names are supplied via a variable, which goes to the code processor, which *should* be pulling the translated fluid name already.

Reply:
Correction, the fluid name translation will take effect in the next update.